README.md

```diff
@@ -1,5 +1,41 @@
 Android TV Samples Repository
 =============================
 
-This repository contains a set of individual Android Studio projects to help you get
-started writing Android TV apps.
+This repository contains a set of individual Android Studio projects to help you get started writing Android TV apps.
```

Comment:
s/Android Studio projects/Android TV projects

These projects do not require android studio to build/compile/run.

Reply:
That's true; I'll update this too